bundles/geocatalog/src/main/java/org/orbisgis/geocatalog/impl/Catalog.java

```diff
@@ -485,6 +487,30 @@ private void createPopupActions() {
                     I18N.tr("Read the content of the database"),
                     GeocatalogIcon.getIcon("refresh"),EventHandler.create(ActionListener.class,
                     this,"refreshSourceList"),KeyStroke.getKeyStroke("ctrl R")).setLogicalGroup(PopupMenu.GROUP_OPEN));
+
+            //Popup:Tools
+            if(wpsClient != null) {
```

Comment:
As wps client is a mandatory reference it is always != null
Or change the cardinality of wpsClient to 0,1 (dynamic)